content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -446,8 +445,8 @@ Please note that:
 
 `kubeadm init` create a first bootstrap token, either generated automatically or provided by the
 user with the `--token` flag; as documented in bootstrap token specification, token should be
-saved as secrets with name `bootstrap-token-<token-id>` under `kube-system` namespace.  Please
-note that:
+saved as secrets with name `bootstrap-token-<token-id>` under `kube-system` namespace.
+Please note that:
```

Comment:
nit: I think you need one more new line space here to get this to appear on the next line, similar to the other "Please note that:" sections on the page. Right now it's still showing in the preview on the same line as the paragraph above. 

Its a small thing, the changes all look good

Reply:
Good suggestion, I have updated, please review